packages/ens-utils/src/ensname.ts

```diff
@@ -326,6 +326,19 @@ export function getRegistrationPotential(name: ENSName): RegistrationPotential {
   }
 }
 
+/**
+ * Splits a label into its individual characters.
```

Comment:
```suggestion
 * Splits a string into its individual characters.
```

This function is for any string, not just for the subset of strings that are labels.